[2022-01-11 15:33:14,767] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: stocks_dag.sql_connect_ manual__2022-01-11T15:33:12.499490+00:00 [queued]>
[2022-01-11 15:33:14,777] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: stocks_dag.sql_connect_ manual__2022-01-11T15:33:12.499490+00:00 [queued]>
[2022-01-11 15:33:14,779] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 15:33:14,779] {taskinstance.py:1239} INFO - Starting attempt 1 of 4
[2022-01-11 15:33:14,780] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 15:33:14,792] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): sql_connect_> on 2022-01-11 15:33:12.499490+00:00
[2022-01-11 15:33:14,797] {standard_task_runner.py:52} INFO - Started process 811 to run task
[2022-01-11 15:33:14,801] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'stocks_dag', 'sql_connect_', 'manual__2022-01-11T15:33:12.499490+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/stocks_dag.py', '--cfg-path', '/tmp/tmpa3l02fbp', '--error-file', '/tmp/tmp6pmzgshs']
[2022-01-11 15:33:14,803] {standard_task_runner.py:77} INFO - Job 133: Subtask sql_connect_
[2022-01-11 15:33:14,860] {logging_mixin.py:109} INFO - Running <TaskInstance: stocks_dag.sql_connect_ manual__2022-01-11T15:33:12.499490+00:00 [running]> on host 36edd51a2216
[2022-01-11 15:33:14,914] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=mcgaritym@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=stocks_dag
AIRFLOW_CTX_TASK_ID=sql_connect_
AIRFLOW_CTX_EXECUTION_DATE=2022-01-11T15:33:12.499490+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-11T15:33:12.499490+00:00
[2022-01-11 15:33:14,959] {python.py:175} INFO - Done. Returned value was: Stocks Database Created
[2022-01-11 15:33:14,981] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=stocks_dag, task_id=sql_connect_, execution_date=20220111T153312, start_date=20220111T153314, end_date=20220111T153314
[2022-01-11 15:33:15,016] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-11 15:33:15,057] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
